src/components/FilterBar.js

```diff
@@ -22,7 +23,8 @@ const types = {
   nuget: nuget,
   git: git,
   crate: crate,
-  pod: pod
+  pod: pod,
+  debian: debian
```

Comment:
The path is going to be deb/debian/... Not sure if it should be deb here.

Reply:
This is only used to get the logo, so it wont be involved by the path structure here